src/common/tiles/BigTile/styled.js

```diff
@@ -0,0 +1,245 @@
+  }
+`;
+
+export const Image = styled.img`
+  width: 312px;
+  height: 464px;
+  border-radius: ${({ theme }) => theme.radius.small};
+  grid-row: span 2;
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    width: 215px;
+    height: 319px;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    width: 114px;
+    height: 169px;
+  }
+`;
+
+export const TileContainer = styled.div`
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    grid-row: span 2;
+  }
+`;
+
+export const Title = styled.header`
+  font-weight: 600;
+  font-size: 36px;
+  color: ${({ theme }) => theme.colors.black};
+  margin: 0 0 24px 0;
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    font-size: 24px;
+    margin: 0 0 18px 0;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    font-size: 13px;
+    margin: 0 0 8px 0;
+  }
+`;
+
+export const Year = styled.p`
+  font-weight: 400;
+  font-size: 22px;
+  color: ${({ theme }) => theme.colors.black};
+  margin: 0 0 24px 0;
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    font-size: 18px;
+    margin: 0 0 18px 0;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    color: ${({ theme }) => theme.colors.darkGrey};
+    font-size: 13px;
+    margin: 0 0 8px 0;
+  }
+`;
+
+export const SubHeaderContainer = styled.div`
+  display: grid;
+  gap: 8px;
+  margin: 0 0 24px 0;
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    margin: 0 0 18px 0;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    color: ${({ theme }) => theme.colors.darkGrey};
+    margin: 0 0 8px 0;
+  }
+`;
+
+export const SubHeader = styled.div`
+  display: grid;
+  grid-template-columns: auto 1fr;
+  gap: 10px;
+`;
+
+export const SubHeaderTitle = styled.p`
+  font-weight: 400;
+  font-size: 18px;
+  color: ${({ theme }) => theme.colors.stormGray};
+  margin: 0;
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    display: none;
+  }
+`;
+
+export const SubHeaderInformation = styled.p`
+  font-weight: 400;
+  font-size: 18px;
+  color: ${({ theme }) => theme.colors.black};
+  margin: 0;
+  line-height: 120%;
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    font-size: 15px;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    font-size: 12px;
+  }
+`;
+
+export const GenreContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 16px;
+  align-items: center;
+  margin: 0 0 24px 0;
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    margin: 0 0 18px 0;
+    gap: 12px;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    margin: 0 0 8px 0;
+    gap: 8px;
+  }
+`;
+
+export const Genre = styled.p`
+  font-weight: 400;
+  font-size: 14px;
+  color: ${({ theme }) => theme.colors.woodsmoke};
+  background-color: ${({ theme }) => theme.colors.mystic};
```

Comment:
Nie wyświetlają się ramki, prawdopodobnie jest to spowodowane brakiem koloru mystic w theme